consensus/src/sync/live/state_queue/live_sync.rs

```diff
@@ -259,4 +259,8 @@ impl<N: Network> LiveSyncQueue<N> for StateQueue<N> {
     fn state_complete(&self) -> bool {
         self.start_key.is_complete()
     }
+
+    fn resolve_block(&mut self, request: crate::consensus::ResolveBlockRequest<N>) {
```

Comment:
I'd suggest to add the respective `use`
```suggestion
    fn resolve_block(&mut self, request: ResolveBlockRequest<N>) {
```